code/modules/projectiles/guns/projectile/automatic/bren.dm

```diff
@@ -5,7 +5,7 @@
 	icon = 'icons/obj/guns/projectile/bren.dmi'
 	icon_state = "bren"
 	item_state = "bren"
-	w_class = ITEM_SIZE_HUGE
+	w_class = ITEM_SIZE_BULKY
```

Comment:
Personally think the bren is over-valued as a simple omni or sts tends to be best.
Im alright with reducing the size